rrule-wasm/src/lib.rs

```diff
@@ -0,0 +1,19 @@
+use rrule::RRuleSet;
+use wasm_bindgen::prelude::*;
+
+/**
+  Get all recurrences of the rrule!
+     rule_set_string: List of rrules
+     limit: Limit must be set in order to prevent infinite loops
+*/
+#[wasm_bindgen]
+pub fn get_all_date_recurrences(rule_set_string: &str, limit: Option<u16>) -> Vec<JsValue> {
```

Comment:
Please don't include the type in variable names. `rule_set_string` -> `rrule_set`